client/landing/stepper/declarative-flow/site-setup-flow.ts

```diff
@@ -185,13 +185,24 @@ export const siteSetupFlow: Flow = {
 
 					return navigate( 'processing' );
 
+				case 'patternAssembler':
+					return navigate( 'processing' );
+
 				case 'processing': {
 					const processingResult = params[ 0 ] as ProcessingResult;
 
 					if ( processingResult === ProcessingResult.FAILURE ) {
 						return navigate( 'error' );
 					}
 
+					// End of Pattern Assembler flow
+					if (
+						isEnabled( 'signup/design-picker-pattern-assembler' ) &&
+						( selectedDesign as Design )?.slug === 'blank-canvas-blocks'
```

Comment:
[nit] It might be better to use `isBlankCanvasDesign` from `@automattic/design-picker`